release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -2986,6 +2986,46 @@ The default `IngressController` CR is configured with 4 HAProxy threads. If you
 
 * For {sno-caps} on {product-version} and Google Cloud Platform (GCP), there is a known issue with the Cloud Network Config Controller (CNCC) entering a `CrashLoopBackOff` state. This occurs at initialization time when the CNCC tries to reach the GCP internal load balancer address and the resulting hairpin traffic is not correctly prevented in OVN-Kubernetes shared gateway mode on GCP causing it to get dropped. Cluster Network Operator will show a `Progressing=true` status in such case. Currently, there is no workaround for this issue. (link:https://issues.redhat.com/browse/OCPBUGS-20554[*OCPBUGS-20554*])
 
+* Large latency spikes may be seen whenever a container is started, on an {sno-caps} with guaranteed CPUs and Interrupt Request (IRQ) load balancing is disabled. (link:https://issues.redhat.com/browse/OCPBUGS-22901[*OCPBUGS-22901*])
+
+* When deploying an application with a large number of pods, some of which have CPU limits configured, the deployment can sometimes fail. The workaround is to re-deploy the application. (link:https://issues.redhat.com/browse/RHEL-7232[*RHEL-7232*])
+
+* On an {sno-caps} with disabled capabilities, the `openshift-controller-manager-operator` may continuously get restarted.  The workaround is to enable the build capability or to manually create the `builds.config.openshift.io` CRD.
++
+Perform the following steps to manually create the `builds.config.openshift.io` CRD:
++
+--
+. Extract the release manifests:
```

Comment:
Recall the "run the following command" language required by the repo contribution guidelines.